### ISSUE TITLE
Name is not associated with the checkboxes, and same announcement “checkbox not checked select row” is being announced for every checkbox.

### ISSUE THREAD

**Issue by huangzumeihan** [microsoft/fluentui#19226]
Actual Result:
Name is not associated with the checkboxes, and same announcement “checkbox not checked select row” is being announced for every checkbox

Expected Result:
Name should be associated with the checkboxes, so screen reader is announce name with checkboxes.

![image](https://user-images.githubusercontent.com/31198399/127944799-6c6bdbcb-6b87-4789-9c66-8f6617745d19.png)

**Comment by smhigley** [comment]
@huangzumeihan if you define one of your columns to function as rowheaders, the checkbox will be named by that column's cell. You can see that in the main DetailsList example, if you turn on selection: https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist

Another option is to create your own custom checkbox render function. Otherwise we don't have anything to go off of to name the checkboxes more specifically than "select row".

Does that solve your naming issue?

**Comment by huangzumeihan** [comment]
@smhigley Thank you very much. I will revise it according to your suggestion

**Comment by smhigley** [comment]
Awesome, glad I could help! I'm closing this for now, but feel free to re-open or create a new bug if you run into an issue with those suggestions :)
